module_utils/network_lsr/argument_validator.py

```diff
@@ -800,6 +804,38 @@ def __init__(self):
         )
 
 
+
+class ArgValidator_DictEthtoolSetring(ArgValidatorDict):
+    def __init__(self):
+        ArgValidatorDict.__init__(
+            self,
+            name="set-ring",
+            nested=[
+                ArgValidatorNum(
+                    "rx", val_min=0, val_max=UINT32_MAX, default_value=None
+                ),
+                ArgValidatorNum(
+                    "rx_mini", val_min=0, val_max=UINT32_MAX, default_value=None
+                ),
+                ArgValidatorNum(
+                    "rx_jumbo", val_min=0, val_max=UINT32_MAX, default_value=None
+                ),
+                ArgValidatorNum(
+                    "tx", val_min=0, val_max=UINT32_MAX, default_value=None
+                ),
+            ],
+        )
+        self.default_value = dict(
+            [(k, v.default_value) for k, v in self.nested.items()]
+        )
+
+
+
+
+
+
```

Comment:
Please, remove this empty lines, they are not needed. Thanks

Reply:
Empty line removed.